perfkitbenchmarker/package_managers.py

```diff
@@ -115,11 +115,39 @@ def GetServiceName(self, package_name):
     """
     pass
 
+  def SetupProxy(self):
+    """
+        This function setup proxy if your cloud is currently installed behind proxy server
+    """
+    env_file = "/etc/environment"
+
+    commands = []
+
+    add_proxy_conf = "sudo touch %s" % env_file
```

Comment:
This shouldn't be needed since `tee` also creates its destination if it doesn't exist yet. Currently, this also has the side effect of always touching the file even if it isn't modifying it.